### ISSUE TITLE
tizonia exiting (OMX_ErrorInsufficientResources) on Fedora 29, LXQT i386

### ISSUE THREAD

**Issue by molo76** [tizonia/tizonia-snap#5]
Fedora 29, pulse-audio works for VLC. 

Installed via Snap, all good. Playing anything from Spotify results in: 

`[mj@fed29-nc10 ~]$ tizonia --spotify-album 'Erpland'
tizonia 0.18.0. Copyright (C) 2019 Juan A. Rubio
This software is part of the Tizonia project <http://tizonia.org>

[Spotify] [Connecting] : 'username'.
[Spotify] [Album search] 'Erpland'.
[Spotify] [Album] 'Erpland'.
[Spotify] [Track] 'Eternal Wheel' [Ozric Tentacles].
[Spotify] [Track] 'Toltec Spring' [Ozric Tentacles].
[Spotify] [Track] 'Tidal Convergence' [Ozric Tentacles].
[Spotify] [Track] 'Sunscape' [Ozric Tentacles].
[Spotify] [Track] 'Mysticum Arabicola' [Ozric Tentacles].
[Spotify] [Track] 'Crackerblocks' [Ozric Tentacles].
[Spotify] [Track] 'The Throbbe' [Ozric Tentacles].
[Spotify] [Track] 'Erpland' [Ozric Tentacles].
[Spotify] [Track] 'Valley Of A Thousand Thoughts' [Ozric Tentacles].
[Spotify] [Track] 'Snakepit' [Ozric Tentacles].
[Spotify] [Track] 'Iscence' [Ozric Tentacles].
[Spotify] [Track] 'A Gift Of Wings' [Ozric Tentacles].
[Spotify] [Tracks in queue] '12'.
[Spotify] [Cache]: '/home/mj/snap/tizonia/common/tizonia-martyn-spotify-elliotmademedoit'
[Spotify] [Login] 'username' logged in.
[Spotify] [Connected] : 'Erpland'.

tizonia exiting (OMX_ErrorInsufficientResources).

 [OMX.Aratelia.audio_renderer.pulseaudio.pcm:port:0]
 [OMX_ErrorInsufficientResources]
`
Username above is in place of my actual Spotify username. 

I read a previous issue about non-standard window managers possibly being a cause, and this is a lxqt install of Fedora.

**Comment by tizonia** [comment]
Hi!, thanks for taking the time to report issues.

Yes, there are problems with snap confinement and Pulseaudio in some non-mainstream window managers. 

> Fedora 29, pulse-audio works for VLC.

So have you installed VLC as a snap or with the native package manager?

**Comment by molo76** [comment]
Yeah, VLC is installed as a native package. I tried again with xfce fresh install and get the same. Laptop is old and can't handle gnome3....

I may try VLC as a snap.

**Comment by loeschzwerg** [comment]
Same issue under Manjaro with i3.
I also tried to use alsa as  default-audio-renderer, but that produced the same error.
I tried to avoid building from AUR because of the long compile time, but I will do now.

**Comment by tizonia** [comment]
@loeschzwerg 

Thanks a lot for the feedback. 

There is some info here on how to verify if the PulseAudio denials and a link to the Snapd Launchpad bug tracker. We could ping the launchpad issue in case the snap of Tizonia on i3 is suffering from the same issue as in Awesome VM (and possibly LXQT).

https://github.com/tizonia/tizonia-snap/issues/6#issuecomment-483995001

**Comment by loeschzwerg** [comment]
`FileNotFoundError: [Errno 2] No such file or directory: '/var/log/syslog'`

I tried the proposed `sudo journalctl ...`, but it didn't return anything, while running next to tizonia.
